package.json

```diff
@@ -42,7 +42,10 @@
     "raw-loader": "0.5.1",
     "react": "16.1.0",
     "react-dom": "16.1.0",
+    "react-redux": "^5.0.7",
```

Comment:
@jaebradley @arizzitano , I was getting quality errors until I ran ```npm i -S react-redux``` as suggested by the linter, but it looks like that causes us to use a ^range in our package.json, is this fine for now or should I be doing ```npm i --save-exact react-redux```